app/controllers/application_controller.rb

```diff
@@ -5,4 +5,14 @@ class ApplicationController < ActionController::Base
   respond_to :html
 
   protect_from_forgery with: :exception
+
+  private
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by(id: session[:user_id])
+    end
+  end
+
```

Comment:
Probably don't need a line between these

Reply:
+1

Reply:
removed the line in sha: 341d0a4